Viewer/Web-AAR-Viewer.html

```diff
@@ -18,31 +18,31 @@
 				display: inline;
 				font-size: 12px;
 			}
-			
+
 			#header-status-text {
 				padding: 0px 20px;
 			}
 		</style>
 	</head>
 	<body>
 		<div class="panzoom"></div>
-		
+
 		<div id="player-line">
 			<button id="player-step-backward" onClick="reportPrevStep()">Prev second</button>
 			<button id="player-step-play" onClick="playReportAuto()">Play report</button>
 			<button id="player-step-forward" onClick="reportNextStep()">Next second</button>
 			<select name="player-speed" id="player-speed">
 				<optgroup label="Play speed">
-					<option value="1">x1</option>
-					<option value="2">x2</option>
-					<option value="5">x5</option>
-					<option value="10">x10</option>
-					<option value="50">x50</option>
+					<option value="1">x0.5</option>
+					<option value="2">x1</option>
+					<option value="10">x5</option>
+					<option value="20">x10</option>
+					<option value="100">x50</option>
```

Comment:
Actually these values are not reflect that value=1 is half-speed, 2 - normal speed and so one, so I changed them to correct values